qa/rpc-tests/replace-by-fee.py

```diff
@@ -63,8 +63,16 @@ def make_utxo(node, amount, confirmed=True, scriptPubKey=CScript([1])):
 
     # If requested, ensure txouts are confirmed.
     if confirmed:
-        while len(node.getrawmempool()):
+        mempool_size = len(node.getrawmempool())
+        while mempool_size > 0:
             node.generate(1)
+            new_size = len(node.getrawmempool())
+            if (mempool_size == new_size):
+                # We might have stuck transactions; assume that
+                # the transactions we needed mined were mined, and
+                # don't worry about what is left.
+                break
+            mempool_size = new_size
```

Comment:
Fair enough -- I ran into an issue where the really large transaction created in one of the tests was too big to mined, but I should be able to rework that test and make this check error out instead.